stock_inventory_exclude_sublocation/__manifest__.py

```diff
@@ -6,7 +6,7 @@
     "name": "Stock Inventory Exclude Sublocation",
     "summary": "Allow to perform inventories of a location without including "
     "its child locations.",
-    "version": "12.0.1.0.1",
+    "version": "13.0.1.0.1",
     "development_status": "Mature",
     "author": "Eficent, " "Tecnativa, " "Odoo Community Association (OCA)",
```

Comment:
Could you remove unnecessary quotes, change Eficent to ForgeFlow and also update the Copyright note?

Reply:
Done!